make/proto.mk

```diff
@@ -39,4 +62,4 @@ endif
 	buf lint $(ROOT_DIR)
 
 .PHONY: proto.gen.all
-proto.gen.all: proto.gen.code proto.gen.doc
\ No newline at end of file
+proto.gen.all: proto.gen.init proto.gen.code proto.gen.doc
```

Comment:
```suggestion
proto.gen.all: proto.gen.code proto.gen.doc
```
We don't have to run `proto.gen.init` before `proto.gen.code` since `proto.gen.code` use docker image to run the tools

Reply:
So when do we use the protocol init?

Reply:
> So when do we use the protocol init?

Emmmm when you don't want to use the docker image, which is not very often

Reply:
OK got it.